proxyd/backend.go

```diff
@@ -220,6 +221,12 @@ func WithConsensusSkipPeerCountCheck(skipPeerCountCheck bool) BackendOpt {
 	}
 }
 
+func WithConsensusForcedCandidate(forcedCandidate bool) BackendOpt {
+	return func(b *Backend) {
+		b.forcedCandidate = forcedCandidate
+	}
+}
```

Comment:
Allows a backend to opt-out of all the other consensus health checks (sync, peers, healthy, banned, etc). This is important for us as we only have a single backend and we never want it to not be a candidate.